src/documents/forms/NewUploadForm.js

```diff
@@ -10,11 +10,12 @@ import {
   EnterDetailsStep,
```

Comment:
Just going to leave this comment here. When I go through each step, it changes the step number. So if there are 4 steps, I expect to complete each step and get to the last one and see 4 for step 4. But each time I complete a step the number of steps change so when I get to the last step, it says 1 New Experience: 

![image](https://user-images.githubusercontent.com/10283712/134694652-5a4b5330-ef73-4546-a2a4-75dc3ff86fb4.png)